src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/variables.scss

```diff
@@ -0,0 +1 @@
+$viewPadding: 60px;
```

Comment:
I am not sure about this... I think it is a nice idea, to keep the padding consistent. I am only not sure if this should be part of another SCSS file, or if we should introduce a `variables.scss` as I have done here.

Reply:
ack